arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/StackFragment.java

```diff
@@ -109,7 +109,9 @@ private StackFragment(
           hub.currentFrame().frame().getStackItem(0).toUnsignedBigInteger();
       final BigInteger codeSize = BigInteger.valueOf(hub.currentFrame().code().getSize());
 
-      boolean prospectivePcNewIsInBounds = codeSize.compareTo(prospectivePcNew) > 0;
+      boolean noOutOfGasException = !Exceptions.outOfGasException(exceptions);
+      boolean prospectivePcNewIsInBounds =
+          codeSize.compareTo(prospectivePcNew) > 0 && noOutOfGasException;
```

Comment:
Genuine mistake: O`O`GX has precedence in exception testing for `JUMP(I)` instructions over `JUMPX`